Controllers/users.js

```diff
@@ -0,0 +1,84 @@
+require('dotenv').config({path: __dirname + '/process.env'})
+database = require("../database");
+var db = database.getdb();
+let jwt = require("jsonwebtoken");
+let middleware = require("../middleware");
+
+async function loginUser(req,res){
+  var username = req.body.username;
+  var password = req.body.password;
+
+  var data = {
+      "username": username,
+      "password":password,
+  }
+
+  try {
+    console.log({username:data.username,password:data.password});
+    const result = await db.collection('details').findOne({username: data.username, password: data.password});
+
+    if (result){
+      let token = jwt.sign({username: data.username},
+        process.env.SECRET,
+        {
+          expiresIn: '24h'
+        }
+      );
+
+    res.status(200).json({
+      success: true,
+      message: 'Authentication successful!',
+      token: token,
+      id: result._id
+      })
+    console.log("Successful login.");
+    }
+
+  else{
+    res.status(404).send("Failed login");
```

Comment:
In the case with wrong password/credentials, the status code should be 401 Unauthorized